commons/com.b2international.index/src/com/b2international/index/revision/DefaultRevisionIndex.java

```diff
@@ -301,7 +300,7 @@ public void removeHook(Hook hook) {
 	 * @return
 	 */
 	List<Hooks.Hook> getHooks() {
-		return ImmutableList.copyOf(hooks);
+		return List.copyOf(hooks);
```

Comment:
This conversion is fine, but there is a "fun" difference between `ImmutableSet#copyOf` and `Set#copyOf` &ndash; the latter uses hashing and iteration order is not consistent.